### PR TITLE
Add optional parameter to set handle-rst-during-fin-rcv

### PR DESCRIPTION
**Pull request by jfrantisek**

### CODE REVIEW

**Review by jirkaluska — APPROVED** [review]

File: setup.sh [hunk 1]
```diff
@@ -11,10 +12,16 @@ while [ $# -gt 0 ]; do
...
+      HANDLE_RST_DURING_FIN_RCV="${1#*=}"
+      ;;
     *)
       printf "************************************************************\n"
       printf "* Error: Invalid argument '${1}'.\n"
-      printf "* Usage: setup.sh [--pool6=<IPv6>] [--lowest-ipv6-mtu=<mtu>]\n"
+      printf "* Usage: setup.sh [options...]\n"
+      printf "*        --pool6=<IPv6>\n"
+      printf "*        --lowest-ipv6-mtu=<mtu>\n"
+      printf "*        --handle-rst-during-fin-rcv=<boolean>\n"
```
Comment (laura-zelenku) [inline setup.sh:24]: I would rather don't accept value for this. If this option is used, just switch `HANDLE_RST_DURING_FIN_RCV` to true.
```suggestion
      printf "*        --handle-rst-during-fin-rcv\n"
```

File: setup.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 POOL6="64:ff9b::/96"
 LOWEST_IPV6_MTU=1280
+HANDLE_RST_DURING_FIN_RCV=false
```
Comment (laura-zelenku) [inline setup.sh:5]: shouldn't be `false` as string? -> "false"?

File: setup.sh [hunk 3]
```diff
@@ -2,19 +2,26 @@
...
+HANDLE_RST_DURING_FIN_RCV="false"
 
 while [ $# -gt 0 ]; do
   case "$1" in
     --pool6=*)
       POOL6="${1#*=}"
       ;;
     --lowest-ipv6-mtu=*)
-      LOWEST_IPV6_MTU="${1#*=}"
+      LOWEST_IPV6_MTU="true"
```
Comment (psiman-wandera) [inline setup.sh]: This doesn't seem right. Shouldn't it be set to `"${1#*=}"`?

File: setup.sh [hunk 4]
```diff
@@ -2,19 +2,26 @@
...
+      ;;
+    --handle-rst-during-fin-rcv=*)
```
Comment (psiman-wandera) [inline setup.sh]: Probably shouldn't contain `=`
```suggestion
    --handle-rst-during-fin-rcv*)
```

File: setup.sh [hunk 5]
```diff
@@ -2,19 +2,26 @@
...
+      HANDLE_RST_DURING_FIN_RCV="${1#*=}"
```
Comment (psiman-wandera) [inline setup.sh]:
```suggestion
      HANDLE_RST_DURING_FIN_RCV="true"
```

**Review by psiman-wandera — CHANGES_REQUESTED** [review]

**Review by laura-zelenku — APPROVED** [review]

**Review by jirkaluska — APPROVED** [review]

**Pull request merged**